modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/exception/mapper/InvalidPaginationExceptionMapper.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.internal.jaxrs.exception.mapper;
+
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+import com.liferay.portal.vulcan.pagination.InvalidPaginationException;
+
+/**
+ * @author Alberto Javier Moreno Lage
+ */
+public class InvalidPaginationExceptionMapper
+	extends BaseExceptionMapper<InvalidPaginationException> {
+
+	@Override
+	protected Problem getProblem(
+		InvalidPaginationException invalidPaginationException) {
+
+		return new Problem(invalidPaginationException);
```

Comment:
I think this will use the exception message in the json response, which is not localized.